pandas/tests/arrays/categorical/test_operators.py

```diff
@@ -186,6 +186,18 @@ def test_comparison_with_unknown_scalars(self):
         tm.assert_numpy_array_equal(cat != 4,
                                     np.array([True, True, True]))
 
+    def test_comparison_with_known_scalars(self):
+        # https://github.com/pandas-dev/pandas/issues/26504
+        # and following comparisons with scalars not in categories should raise
+        # for unequal comps, but not for equal/not equal
+        cat = Categorical([1, 2, 3, None], categories=[1, 2, 3], ordered=True)
+
+        msg = ("Cannot compare a Categorical for op __{}__ with a scalar,"
+               " which is not a category")
```

Comment:
What this `msg` variable for?

Reply:
I was trying to return some error messages and got interrupted by my coworkers. Basically, it should not be there.